workflows/sample/inputs/docker.inputs.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sample.pb_conda_image": "projectastro.azurecr.io/pb4:latest",
+  "sample.deepvariant_image": "google/deepvariant:1.0.0",
```

Comment:
`google/deepvariant:1.1.0`

Reply:
Also, if you really want to future proof this, use `google/deepvariant:1.1.0-gpu` and this will work on GPUs if available (assuming `singularity exec --nv`).  There's little downside.

Reply:
deepvariant now changed to gpu version